src/cpp/src/whisper_pipeline_static.cpp

```diff
@@ -394,7 +394,17 @@ void add_attention_mask_input(std::shared_ptr<ov::Model> model) {
     pm.run_passes(model);
 }
 
-void reshape_to_static(std::shared_ptr<ov::Model> model, const uint32_t input_size, const uint32_t kvcache_size) {
+
+uint32_t get_encoder_hidden_state_size(const std::shared_ptr<ov::Model>& encoder) {
+    for (auto output : encoder->outputs()) {
+        if (output.get_any_name() == "last_hidden_state") {
+            return output.get_partial_shape()[1].get_length();
```

Comment:
Need to get both dimension 1 and dimension 2 of encoder output. Dim 1 will be 1500. Dim 2 will vary based on model version. For example, dim 2 = 512 for base, 768 for small.